Integrations/Flashpoint/Flashpoint.py

```diff
@@ -0,0 +1,1321 @@
+                        'detail', 'N/A'))
+            elif status_code == 401:
+                raise ValueError(
+                    "Encountered error while trying to get information from Flashpoint: Invalid API Key is "
+                    "configured")
+            elif status_code == 404:
+                raise ValueError("No record found for given argument(s): Not Found")
+            elif status_code in (521, 403):
+                raise ValueError("Test connectivity failed. Please provide valid input parameters.")
+            else:
+                resp.raise_for_status()
+
+        return resp_json
+
+
+''' HELPER FUNCTIONS '''
+
+
+def get_apikey():
+    """ Get API Key from the command argument"""
+    api_key = demisto.params()["api_key"]
+
+    return api_key
+
+
+def get_url_suffix(query):
+    """
+    Create url-suffix using the query value with url encoding
+
+    :param query: value of query param
+    :return: url-encoded url-suffix
+    """
+    return r'/indicators/simple?query=' + urllib.parse.quote(query.encode('utf8'))
+
+
+def get_events_from_ioc_resp(indicators):
+    """
+    Extract event details and href values from each of the indicator in an indicator list
+
+    :param indicators: list of indicators
+    :return: dict containing event details and href
+    """
+    events = []
+    hrefs = []
+    for indicator in indicators:
+        hrefs.append(indicator.get('Attribute', {}).get('href', ''))
+
+        event = indicator.get('Attribute', {}).get('Event', {})
+
+        tags_value = ''
+        for tag in event['Tags']:
+            tags_value = tags_value + tag + ', '
+        if tags_value:
+            tags_value = tags_value[:-2]
+
+        observed_time = time.strftime('%b %d, %Y  %H:%M', time.gmtime(float(event['timestamp'])))
+
+        events.append({
+            'Date Observed (UTC)': observed_time,
+            'Name': event.get('info', ''),
+            'Tags': tags_value,
+        })
+
+    return {'events': events, 'href': hrefs}
+
+
+def convert_event(client, event, fpid):
+    """
+    Prepare required event jason object from event response
+
+    :param client:
+    :param event: event indicator from response
+    :return: required event json object
+    """
+    observed_time = time.strftime('%b %d, %Y  %H:%M', time.gmtime(float(event['timestamp'])))
+    name = event.get('info', '')
+    uuid = event.get('uuid', '')
+    if uuid:
+        fp_link = client.url + '/home/technical_data/iocs/items/' + uuid
+        name_str = '[{}]({})'.format(name, fp_link)
+    else:
+        name_str = name
+
+    tags_value = ''
+    for tag in event.get('Tag', []):
+        tags_value = tags_value + tag.get('name', '') + ', '
+    if tags_value:
+        tags_value = tags_value[:-2]
+
+    event_creator_email = event.get('event_creator_email', '')
+
+    event = {
+        'Observed time (UTC)': observed_time,
```

Comment:
avoid using spaces and special chars in the output keys.

Reply:
Removed whitespaces and special chars from output keys